.github/workflows/npm-prerelease.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish Pre-release to npm
+
+on:
+  push:
+    branches:
+      - "release/v*"
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "20.x"
+          registry-url: "https://registry.npmjs.org/"
+
+      - name: Install Dependencies
+        run: pnpm install
+
+      - name: Run Tests
+        run: pnpm test
```

Comment:
If this step fails will it stop the release or will the step afterwards still run? I don't think we want to do a release if the tests fail.

Reply:
We should probably also run these tests via Docker, like we do for the test action.

Reply:
> If this step fails will it stop the release or will the step afterwards still run? I don't think we want to do a release if the tests fail.

 If the step fails, the workflow stops here and does not proceed to the following steps.